README.md

```diff
@@ -70,14 +71,18 @@ Usage: migration-utils [-hrVx] [--debug] -a=<targetDir> [-d=<f3DatastreamsDir>]
   -a, --target-dir=<targetDir>
                              Directory where OCFL storage root and supporting
                                state will be written
-  -m, --migration-type=<migrationType>  Type of OCFL objects to migrate to. Choices: 
+  -m, --migration-type=<migrationType>
```

Comment:
It looks like you edited this file by-hand. I would suggest having the "help" display in your terminal, then pasting the output directly over this block of the README.